apps/src/applab/applab.js

```diff
@@ -62,7 +62,11 @@ import {
 } from '../lib/tools/jsdebugger/redux';
 import JavaScriptModeErrorHandler from '../JavaScriptModeErrorHandler';
 import * as makerToolkit from '../lib/kits/maker/toolkit';
-var project = require('@cdo/apps/code-studio/initApp/project');
+import project from '../code-studio/initApp/project';
+import * as thumbnailUtils from '../util/thumbnail';
+
+// Needed by html2canvas in SVGContainer.hasFabric to work on IE 11.
+window.html2canvas = thumbnailUtils.html2canvas;
```

Comment:
it does have to be on window, unfortunately: https://github.com/niklasvh/html2canvas/blob/master/src/svgcontainer.js#L13